src/gentropy/study_locus_validation.py

```diff
@@ -41,9 +41,11 @@ def __init__(
         # Running validation then writing output:
         study_locus_with_qc = (
             StudyLocus.from_parquet(session, list(study_locus_path))
-            .validate_lead_pvalue(
-                pvalue_cutoff=gwas_significance
-            )  # Flagging study locus with subsignificant p-values
+            # Flagging study locus with subsignificant p-values
+            .validate_lead_pvalue(pvalue_cutoff=gwas_significance)
+            # Add flag for MHC region
+            .qc_MHC_region()
+            # Excluding MHC region
```

Comment:
We are not excluding, just flagging at this point.

Reply:
```suggestion
            # Flagging MHC region
```